Remotion/Web/Development.Analyzers.IntegrationTests/Web.Development.Analyzers.IntegrationTests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ProjectType>Test</ProjectType>
+    <UseWebForms>true</UseWebForms>
+  </PropertyGroup>
+
+  <Import Project="..\..\..\Build\Shared.build.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0-windows</TargetFrameworks>
```

Comment:
Just a head's up: right now, we don't run the net5.0 tests on the automated build. Build script must first learn to deal with dotnet test.